### PR TITLE
Assignment2 cmalik

### PR DESCRIPTION
**Pull request by cmalik3595**
- Review comments from previous assignment
- Assignment number 2.

### CODE REVIEW

File: assignment2/baseball_assignment.sql [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+	FROM batter_game_actual_info batters_info1
+       		JOIN batter_game_actual_info batters_info2 
+		ON batters_info1.batter = batters_info2.batter	
+		WHERE batters_info2.game_date BETWEEN batters_info1.100_day_before_date AND DATE(batters_info1.game_date) 
+	GROUP BY batter,game_date
+	ORDER BY batter,game_date;
+
+SELECT '100 day Rolling Average operation completed !!' AS '';
+select now() as 'Completion Time';
+
```
Comment (Anujagagangras) [inline assignment2/baseball_assignment.sql:93]: It is taking longer time to execute rolling average part of the code. Please work on that part to fix it.

**Review by Anujagagangras — CHANGES_REQUESTED** [review]

File: assignment2/baseball_assignment.sql (hunk 1 above)
Comment (Anujagagangras) [reply assignment2/baseball_assignment.sql:93]: you can try rolling average using window function.

Comment (cmalik3595) [reply assignment2/baseball_assignment.sql:104]: Added window function. Thanks for comment.

Comment (cmalik3595) [reply assignment2/baseball_assignment.sql:104]: Separated slower algorithm from the main assignment.

**Review by Anujagagangras — APPROVED** [review]

**Pull request merged**